framework/routing/__init__.py

```diff
@@ -392,12 +413,15 @@ def __init__(self, template_name, renderer=None, error_renderer=None,
                      to add to data from view function
         :param detect_render_nested: Auto-detect renderers for nested
             templates?
+        :param safe: Boolean: turn on markup-safe escaping?
```

Comment:
That's a good question. The names were intentionally made different, because they represent different and opposite behaviors:

- `safe` refers to turning on markupsafe escaping. We want it to be True by default. (so that people who override it are **literally** saying "safe=False")
- `trust` refers to turning *off* markupsafe escaping. We want it to be False by default (people must explicitly indicate that the content can be trusted as raw HTML)

Because the status tuple is just data (not HTML), we judged that the word "safe" was not as meaningful in that different context. Hence the different semantics and defaults. I'm probably overthinking this.